src/app/client/src/app/modules/cbse-program/components/ckeditor-tool/ckeditor-tool.component.ts

```diff
@@ -58,7 +58,8 @@ export class CkeditorToolComponent implements OnInit, AfterViewInit, OnChanges {
   }
   assetConfig: any = {
     'image': {
-      'size': '50'
+      'size': '50',
```

Comment:
This info we have to take it from programs configuration for this component. It helps to support other image types specific to programs & max limit of the image